### ISSUE TITLE
Compilation error

### ISSUE THREAD

**Issue by kupiqu** [dschopf/qalculate#4]
I get a compilation error with last introduced changes:

```
qalculate/plugin/qwrapper.cpp: In member function ‘QString QWrapper::get_exchange_rates_time()’:
qalculate/plugin/qwrapper.cpp:90:25: error: ‘class Calculator’ has no member named ‘getExchangeRatesTime’
   dt.setTime_t(m_pcalc->getExchangeRatesTime());
```

**Comment by dschopf** [comment]
You seem to be using an older version of libqalculate.
This issue should be fixed with 27f7f55318228963d868b036c5c256dc42f2e3ed.

**Comment by kupiqu** [comment]
Yes, 0.9.7 version in Ubuntu.

Thanks!
